main.go

```diff
@@ -135,14 +138,20 @@ func main() {
 				Usage: "size, in pages, of the internal perf ring buffer used to send blobs from the kernel",
 			},
 			&cli.StringFlag{
-				Name:  "output-path",
+				Name:  "output-dir",
 				Value: "/tmp/tracee",
-				Usage: "set output path",
+				Usage: "set output dir",
+			},
+			&cli.BoolFlag{
+				Name:    "clear-output-dir",
+				Aliases: []string{"rm"},
```

Comment:
Using rm as a shortcut may be a bit confusing, don't you think?

Reply:
I changed it to clear, is it better?